transform-plugins/widgets/ValueMapper-transform.json

```diff
@@ -5,10 +5,16 @@
   "configuration-groups": [
     {
       "properties": [
-        {
-          "widget-type": "textbox",
+	    {
+          "name": "mapping",
           "label": "mapping",
```

Comment:
For display purpose, we should have the label to start with Uppercase letter. This applies to line 21 as well